integrations/airflow/tests/integration/integration.py

```diff
@@ -175,19 +179,23 @@ def check_jobs_meta():
     assert len(if_not_exists_job['inputs']) == 0
     assert len(if_not_exists_job['outputs']) == 0
     assert if_not_exists_job['location'] is None
+
+    # TODO: waiting for backend fix
     assert if_not_exists_job['context']['sql'] is not None
-    assert if_not_exists_job['context']['airflow.operator'] == \
-           'airflow.operators.postgres_operator.PostgresOperator'
-    assert if_not_exists_job['context']['airflow.task_info'] is not None
-    assert if_not_exists_job['context']['airflow.version'] == AIRFLOW_VERSION
-    assert if_not_exists_job['context']['marquez_airflow.version'] == MARQUEZ_AIRFLOW_VERSION
+    # TODO: no airflow context data yet
+    # assert if_not_exists_job['context']['airflow.operator'] == \
+    #        'airflow.operators.postgres_operator.PostgresOperator'
+    # assert if_not_exists_job['context']['airflow.task_info'] is not None
+    # assert if_not_exists_job['context']['airflow.version'] == AIRFLOW_VERSION
+    # assert if_not_exists_job['context']['marquez_airflow.version'] == MARQUEZ_AIRFLOW_VERSION
```

Comment:
let's add those to a custom facet.

Reply:
Will add those custom facets in separate PR